---
title: "Research"
layout: textlay
excerpt: "Djordje Bajić -- Research"
sitemap: false
permalink: /research/
---


# Research


We seek to predictively understand how microbial communities assemble, function, and respond to environmental change. Below is a glimpse of some of the questions we are currently interested in answering.

<br/><br/>

---


![]({{ site.url }}{{ site.baseurl }}/images/unlock.jpg){: style="width: 500px; float: left; margin: 0px  10px"}

**Understanding the metabolic principles of microbial community assembly.**
The chemical composition of the environment is a critical driver of the species composition of microbial communities. We work towards understanding quantitatively how the composition of microbial communities is determined by their metabolic environment, and how it responds to metabolic perturbation. To this end, we combine experiments in which we assemble of model microbial communities (both in high-throughput and in bioreactors, where we collaborate with the UNLOCK facility), and modeling. Achieving this goal could pave the way to precisely engineering mirobial community composition through the manipulation of their metabolic environment.

[[Science 2023](https://www.science.org/doi/10.1126/science.adg0727)]
[[Cell Systems 2022](https://www.sciencedirect.com/science/article/pii/S2405471221003793)]

<br/><br/>



---


![]({{ site.url }}{{ site.baseurl }}/images/pubpic/electrosynth.png){: style="width: 300px; float: right; margin: 0px  10px"}
**e-COMM: Understanding and engineering electrosynthetic communities.**
As part of the [Zero Emission Biotechnology Program](https://www.tudelft.nl/tnw/over-faculteit/afdelingen/biotechnology/zero-emission-biotechnology-program) at TU Delft, we are steering our expertise to study biofilm microbial communities transforming green electricity and CO2 to valuable products. This project is a collaboration with the groups of [Ludovic Jourdin](https://www.tudelft.nl/tnw/over-faculteit/afdelingen/biotechnology/research-sections/bioprocess-engineering/ludovic-jourdin-group) and [Jean-Marc Daran](https://www.tudelft.nl/tnw/over-faculteit/afdelingen/biotechnology/research-sections/industrial-microbiology/jean-marc-daran-group) in our department.

<br/><br/>
<br/><br/>
<br/><br/>
<br/><br/>
<p style="text-align: right">&copy; Image courtesy of Marika Zegers. </p>

<br/><br/>


---


![]({{ site.url }}{{ site.baseurl }}/images/pubpic/starch_epsilon.png){: style="width: 350px; float: left; margin: 0px  10px"}

**Engineering microbiome function through its emergent properties**
Understanding in detal how microbial species interact within communities to collectively deliver a function represents a colossal task. We collaborate with [Alvaro Sanchez](https://sanchezlaboratory.weebly.com/), [Juan Diaz-Colunga](https://jdiazc9.github.io/), and others, to take an alternative, *statistical* approach. Instead of aiming to understand first all the details, we take a shortcut and "start from the roof" by looking for emergent patterns that can help us engineer communities with a desired output, without necessarily understanding the underlying mechanisms. We also apply these methods in collaboration with researchers in different fields to help them engineer microbial communities with interesting functions, for example or those helping crops grow and resist pathogens ([Ivica Dimkić](https://bio.bg.ac.rs/zaposleni/dimkic-ivica/)) or those fermentating soymilk ([Nemanja Stanisavljević](https://imgge.bg.ac.rs/lat/nemanja-stanisavljevic) and [Nemanja Kljajević](https://imgge.bg.ac.rs/lat/nemanja-kljajevic)). 

[[Cell, 2024](https://www.biorxiv.org/content/10.1101/2022.06.21.496987v1)]
[[Cell Syst., 2023](https://www.cell.com/cell-systems/fulltext/S2405-4712(22)00499-9)]
[[Proc.R. Soc. B, 2023](https://royalsocietypublishing.org/doi/full/10.1098/rstb.2022.0053)]
[[Nat. Comm,, 2021](https://www.nature.com/articles/s41467-021-21844-7)]

<br/><br/>



---

![]({{ site.url }}{{ site.baseurl }}/images/pubpic/comets_code.png){: style="width: 300px; float: right; margin: 0px  10px"}

**New methods: building digital twins of microbial communities.**
Guided by the principle “if you can't build it, you don't understand it”, we are building mechanistic, metagenome-scale microbial community metabolic models. We are part of the developer team of COMETS, a software platform for the "Computation of microbial ecosystems in time and space", in collaboration with [Daniel Segrè](https://www.bu.edu/segrelab/) and [Ilija Dukovski](https://www.bu.edu/segrelab/profile/2007/). 

[[PLoS Comp. Biol. 2023](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011363)]
[[Nature Protocols 2021](https://www.nature.com/articles/s41596-021-00593-3)]
<br/><br/>
<br/><br/>
<br/><br/>
<br/><br/>
<br/><br/>


---


![]({{ site.url }}{{ site.baseurl }}/images/innovations.png){: style="width: 400px; float: left; margin: 0px  10px"}
**Using metabolic models to understand ecology and evolution.** 
We are interested in a broad range of topics in microbial ecology and evolution. We address these use metabolic models as a workhorse to explore empirically calibrated, realistic genotype-phenotype-ecology maps. We are interested in a broad range of topics in microbial ecology and evolution. For example, we are currently collaborating with [Maria Rebolleda-Gómez](https://mrebolleda.github.io/) to study how microbial metabolisms innovate during evolution. 

[[Mol. Biol. Evol. 2023](https://academic.oup.com/mbe/article/40/9/msad187/7250431)]
[[Current Opinion in Biotechnology 2021](https://www.sciencedirect.com/science/article/pii/S0958166919300722)]
[[Frontiers in Microbiology 2021](https://www.frontiersin.org/articles/10.3389/fmicb.2021.718082/full)]

<br/><br/>
<br/><br/>
<br/><br/>
